contrib/groovy.grails/test/unit/src/org/netbeans/modules/groovy/grails/api/GrailsRuntimeTest.java

```diff
@@ -106,7 +106,7 @@ public void testCommandDescriptor() throws IOException {
         desc.getProps().setProperty("wrong", "wrong");
         assertEquals(props, desc.getProps());
 
-        String[] correctArgs = args.clone();
+        String[] correctArgs = new ArrayList<String>.toArray(args);
```

Comment:
this doesn't compile since the `()` is missing. But what worse is that this is wrong. It would create an empty AL and copy `null` into the first element of `args` to mark the end of the (empty) array.

please use `Arrays.copyOf(args, args.length)` for example. Same applies to the other occurrences.